migrations/2_deploy_contracts.js

```diff
@@ -5,7 +5,7 @@ const SampleRecipient = artifacts.require('TestRecipient')
 const Forwarder = artifacts.require('Forwarder')
 
 module.exports = async function (deployer) {
-  await deployer.deploy(StakeManager)
+  await deployer.deploy(StakeManager, 30000)
```

Comment:
you defined a constant for it, didn't you?

(not for this issue, but I thnk "truffle migrate" is MUCH more robust for installing contracts on networks than our deployer tool. 
Environments are great, and we should have one for each network, and that's great way to document the params used in each network. stakeManagerMaxUnstake is just one environment param

Reply:
It cannot be resolved in the 'migrations' script. Migrations are not used anymore but now is not the time to remove those.